ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/HomeScreenMapContainerViewModel.kt

```diff
@@ -122,8 +124,10 @@ internal constructor(
         .toLiveData()
 
     suggestLoiJobs =
-      surveyRepository.activeSurveyFlow.map {
-        it?.jobs?.filter { job -> job.suggestLoiTaskType != null }?.toList() ?: listOf()
+      surveyRepository.activeSurveyFlow.combine(cameraZoomUpdates.asFlow()) { survey, zoomLevel ->
+        if (zoomLevel < CLUSTERING_ZOOM_THRESHOLD) return@combine listOf()
+       
+        survey?.jobs?.filter { job -> job.suggestLoiTaskType != null }?.toList() ?: listOf()
```

Comment:
Will this refresh the list on every zoom level change? Or does it work like `distinctUntilChanged()` in Rx?

Reply:
I don't see any change. Did you forget to push it?

Reply:
Weird I wasn't even seeing it locally, but I swear I made the change. Anyways I made the change again and pushed it now!